config/enums.cwt

```diff
@@ -52,6 +52,267 @@ enums = {
         very_high_ai_values_divergence #  350 #About 85% (46-98%) of characters will be at or below this value for the avarage character
         #Percentages in parentheses are the observed spans
     }
+    enum[event_themes] = {
```

Comment:
As far as I can tell these are defined in game\common\event_themes, so a type definition and a reference to `<event_theme>` should dynamically generate these from vanilla + mod

Reply:
I tried to get `types` to work, but it kept on saying 'unknown type'. Is there an exmple in the code I could take a look at? I tried to find one, but it was harder than I would've thought.

Reply:
https://github.com/cwtools/cwtools-ck3-config/blob/master/config/common/event_themes.cwt
has the definition, and then just `theme = <event_theme>`. With that one line change it seems to work :)
![image](https://user-images.githubusercontent.com/447391/93711147-e0bbb500-fb43-11ea-9f29-2451273f0deb.png)

Reply:
Wow, I haven't been able to get autocomplete like that at all yet with CWTools. Could it be that my game directory isn't right? Mine just autocompletes with any text in the one file I'm working in.

Reply:
Make sure to "open folder", and ensure that "Crusader Kings III" is in the path somewhere.
So if you open the vanilla folder "Crusader Kings III/game" (because it's in `game` now) then open a .txt file, it should just activate.
You know it's working when it says `ck3` bottom right

Reply:
Oh weird. I had to run VS Code as administrator. Now it does work.

Reply:
Okay, now I can fix all these issues.